src/etc/rc

```diff
@@ -71,12 +71,6 @@ while read FS_PART FS_MNT FS_TYPE FS_MORE; do
 	FS_DEV=$(echo ${FS_PART} | awk 'match($0, /^\/dev\/(gpt|ufs)\/.+$/) { print substr( $0, RSTART + 5, RLENGTH - 5 )}')
 	if [ -n "${FS_DEV}" ]; then
```

Comment:
zap

Reply:
what do you mean with zap? ^^

Reply:
all those lines are no longer needed = no code to execute

Reply:
edited PR, will merge now :)